Lib/importlib/_bootstrap_external.py

```diff
@@ -221,280 +221,9 @@ def _write_atomic(path, data, mode=0o666):
 
 _code_type = type(_write_atomic.__code__)
 
+MAGIC_NUMBER = (sys.implementation._pyc_magic_number).to_bytes(2, 'little') + b'\r\n'
```

Comment:
FYI, `sys.implementation` is where we put information for comparing between implementations, rather than info specific to an implementation.  I'd expect `_pyc_magic_number` to live in the `_imp` module.